app/Resources/views/events/invitation.html.twig

```diff
@@ -24,19 +24,19 @@
                 <div class="form__row">
                     {{ form_label(invitation_form.email, 'Votre adresse e-mail') }}
                     {{ form_errors(invitation_form.email) }}
-                    {{ form_widget(invitation_form.email, { 'attr': { 'class': 'form--full' } }) }}
+                    {{ form_widget(invitation_form.email, { 'value': app.user.emailAddress is defined ? app.user.emailAddress, 'attr': { 'class': 'form--full' } }) }}
```

Comment:
Maybe you should create the object in the controller instead, using a static method like `createFromAdherent($this->getUser())` to pre set the data, doing it in the view prevent the user to change it as it would be overridden in case the template is reloaded with errors.